### PR TITLE
Function to save the number of searched by list - #351

### PR DESCRIPTION
**Pull request by moto3z**
## Description

- target issue : https://github.com/fosslight/fosslight/issues/351 
- I'm working on adding a feature for the above issue, but it's difficult to proceed further due to a lack of understanding of this project, so I'd like help.

### Q1 ) Is this what the project leaders want?

- before (As-is)
image

- after (To-Be)
image

### Q2 ) just worry
- Symptom
    -  : It seems to work fine, but I don't know.
- The cause of the issue I expect
    -  : I'm worried because it went well without any problems. A change has occurred in the core domain User class, but I don't know what more work is needed

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: db/initdb.d/fosslight_create.sql [hunk 1]
```diff
@@ -2374,6 +2374,10 @@ INSERT INTO `T2_CODE_DTL` (`CD_NO`, `CD_DTL_NO`, `CD_DTL_NM`, `CD_SUB_NO`, `CD_D
 	('226', '6', 'Specification Constraints', '', '', 6, 'Y'),
 	('226', '7', 'Restricted Redistribution', '', '', 7, 'Y'),
 	('226', '8', 'Commons Clause Restriction', '', '', 8, 'Y'),
+    ('227', '1', '15', '', '', 15,'Y')
```
Comment (soimkim) [inline db/initdb.d/fosslight_create.sql:2377]: @moto3z, `,` 가 없어서 에러 날 것으로 보입니다.

**Review by soimkim — CHANGES_REQUESTED** [review]

**Comment by soimkim** [comment]
@moto3z , db/initdb.d/fosslight_create.sql 수정시에는 https://github.com/fosslight/fosslight/blob/develop/migration/migration/scripts/20230818004358_update_v1.6.1.sql 도 수정해주십시오.

**Comment by soimkim** [comment]
@moto3z , 수정 완료되면 Ready for review 클릭해주십시오.

**Comment by moto3z** [comment]
@soimkim 안녕하세요 소임님! 피드백주신 내용 반영해서 새로운 PR 로 다시 작성해서 만들었습니다.  
(새로운 PR 링크 : https://github.com/fosslight/fosslight/pull/1004)

그래서 혹시 해당 PR 은 close 처리를 부탁드려도 될까요?

**Comment by jongwooo** [comment]
Superseded by #1004.

**Pull request closed without merging**